### PR TITLE
fix: check for margin-left and margin-right not present

### LINKED ISSUE #52181: False positive on Cafe Menu step 73
### Describe the Issue

False positive on Cafe Menu step 73:

```
.item p {
  display: inline-block;
  margin:5px;
}
```

Causes the test to pass, but it's wrong, since it breaks the layout (by also adding a 5px margin to each side).

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-basic-css-by-building-a-cafe-menu/step-73

### Your code

```
.item p {
  display: inline-block;
  margin:5px;
}
```

### Expected behavior

Test should fail

### Screenshots

_No response_

### System

- Device: Laptop
- OS: Windows 10
- Browser: Brave
- Version: 1.59.124 (Chromium 118.0.5993.117)

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by rockypukar**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #52181

In this code update, I have checked to make sure that  margin-left and margin-right are not present so campers cannot use ```margin: 5px``` or ```margin: 5px 0``` or ```margin: 5px 0 5px 0``` which will set margin-left and margin-right. The campers have to compulsorily use ```margin-top: 5px``` and ```margin-bottom: 5px```.

### CODE REVIEW

**Review by Sembauke — APPROVED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f45a276c093334f0f6e9df4.md [hunk 1]
```diff
@@ -48,6 +48,20 @@ const itemPMarginBottom = new __helpers.CSSHelp(document).getStyle('.item p')?.g
 assert(itemPMarginBottom === '5px');
 ```
 
+Your `p` element nested in your `.item` elements should not have `margin-left`.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f45a276c093334f0f6e9df4.md:51]:
```suggestion
Your `p` elements nested in your `.item` elements should not have a `margin-left` set.
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f45a276c093334f0f6e9df4.md [hunk 2]
```diff
@@ -48,6 +48,20 @@ const itemPMarginBottom = new __helpers.CSSHelp(document).getStyle('.item p')?.g
...
+
+```js
+const itemPMarginLeft = new __helpers.CSSHelp(document).getStyle('.item p')?.getPropertyValue('margin-left');
+assert(!itemPMarginLeft);
+```
+
+Your `p` element nested in your `.item` elements should not have `margin-right`.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f45a276c093334f0f6e9df4.md:58]:
```suggestion
Your `p` elements nested in your `.item` elements should not have a `margin-right` set.
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
nit

**Comment by RockyPukar1** [comment]
@naomi-lgbt The requested change has been made.

**Review by naomi-lgbt — APPROVED** [review]

**Comment by naomi-lgbt** [comment]
Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 🎉

**Pull request merged**